tutorials/hdfs-sync/src/main/java/com/datatorrent/apps/copy/HDFSFileCopyApp.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This code is not part of Apache project. But, this is open-sourced under Apache license for any one to use, make changes etc.
This is approved by Amol, Ashwin.